gpkit/interactive/plot_sweep1d.py

```diff
@@ -0,0 +1,48 @@
+"Implements plot_sweep1d function"
+import matplotlib.pyplot as plt
+import numpy as np
+from ..exceptions import InvalidGPConstraint
+from ..tools import autosweep_1d
+
```

Comment:
plot_sweep1d now supports a full constant/freeposy grid, so it'd still be useful to keep around, but I like both of those a lot!